metrics/web/lib/project_groups/presentation/widgets/project_group_card.dart

```diff
@@ -164,11 +164,27 @@ class _ProjectGroupCardState extends State<ProjectGroupCard> {
 
     if (projectGroupsNotifier.projectGroupDialogViewModel == null) return;
 
-    await showDialog(
+    await _showProjectGroupDialog(
       context: context,
       builder: (_) => EditProjectGroupDialog(),
     );
 
     projectGroupsNotifier.resetProjectGroupDialogViewModel();
   }
+
+  /// Shows a project group dialog
+  /// and applies a barrier color from the metrics theme.
+  Future<void> _showProjectGroupDialog({
+    BuildContext context,
+    WidgetBuilder builder,
```

Comment:
Since this is a stateful widget, we can get the build context from the `context` getter of the `State` class, so we should avoid passing the context to the methods. Also, it seems like it will be better to pass the `Widget` instead of `WidgetBuilder`. What do you think?